pvlib/atmosphere.py

```diff
@@ -196,7 +198,12 @@ def get_relative_airmass(zenith, model='kastenyoung1989'):
 
     .. [6] Keith A. Pickering. "The Ancient Star Catalog". DIO 12:1, 20,
 
-    .. [7] Matthew J. Reno, Clifford W. Hansen and Joshua S. Stein, "Global
+    .. [7] C. Gueymard, "Direct solar transmittance and irradiance
+       predictions with broadband models. Part I: detailed theoretical
+       performance assessment". Solar Energy, vol 74, pp. 355-379, 2003.
+       :doi:`10.1016/S0038-092X(03)00195-6`
+
+    .. [8] Matthew J. Reno, Clifford W. Hansen and Joshua S. Stein, "Global
        Horizontal Irradiance Clear Sky Models: Implementation and Analysis"
        Sandia Report, (2012).
```

Comment:
It seems that this reference isn't cited anywhere.

Reply:
https://github.com/pvlib/pvlib-python/issues/837

Reply:
@AdamRJensen That reference was likely added because it provides a concise summary of the air mass models and a comparison among them.